lib/setup.rb

```diff
@@ -1,10 +1,40 @@
 class Setup
-  def initialize(board, ships)
-    @board = board
-    @ships = ships
+  attr_reader :player,
+              :board
+
+  def initialize(board, ships, player)
+    @board  = board
+    @ships  = ships
+    @player = player
+  end
+   # it will be a string concatenation of a sample of A..D and a sample of 1..4
```

Comment:
We should remove all these comments probs

Reply:
Yup! just deleted!